docs/components/SideBar/SideBar.jsx

```diff
@@ -20,6 +20,7 @@ export default function SideBar({className}) {
         <NavLink href="/design/typography">Typography</NavLink>
         <NavLink href="/design/sizing">Sizing</NavLink>
         <NavLink href="/design/page-layout">Page Layout</NavLink>
+        <NavLink href="/design/compound-form">Compound Form</NavLink>
```

Comment:
That's a good idea. I think initially we thought of putting Compound Form as a sub-nav under Page Layout. cc: @CarlNelson -- should we still do this?